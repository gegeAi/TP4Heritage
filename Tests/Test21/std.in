R rec 1 2 3 4
S seg 1 2 3 4
PC poly 1 2 3 4
CLEAR
LIST
EXIT
