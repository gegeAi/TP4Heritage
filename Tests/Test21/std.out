OK
#New object: rec
OK
#New object: seg
OK
#New object: poly
OK
